### ISSUE TITLE
Incomprehensible import issues

### ISSUE THREAD

**Issue by phorward** [QQuick/Transcrypt#559]
Hi there,

I'm having the following, very simple project setup with a tiny library:
![image](https://user-images.githubusercontent.com/16870072/45738787-a4ba8600-bbf1-11e8-8fe9-9106ee0fa124.png)

When I compile this like in the terminal window above, the resulting html-site gives the error message

``
SyntaxError: redeclaration of import __all__ test.js:6:24 note: Previously declared at line 3, column 277 test.js:3:277
``

Then I thought about using the ``#__pragma__("xglob")`` only in the mylib.mylib module. Compiling the project then without the ``--xglobs`` option (compiler command: ``$ transcrypt -n -a --kwargs --opov --tconv test``) gives the html-site giving the error message with

```
TypeError: args[0] is undefined org.transcrypt.__runtime__.js:166:22
```

I don't know what may cause this message. Then I tried to compile the project without any command line parameters, using the command-line ``$ transcrypt -n -a test``, now I get

```
TypeError: mylib.mylibHello is not a function
```

So what is wrong with my setup? Isn't it possible to divide a library in multiple parts and use the ``__init__.py`` to export all necessary symbols to the public? The only way to get it run is changing the ``test.py`` file's import line to read

```python
import mylib.mylib as mylib
```

But this is not an option for a huger library. What to do now? I'm not sure if it is a bug or something I'm doing wrong, or maybe both? The project runs without any problems in CPython3.

**Comment by JdeH** [comment]
You're not doing anything wrong.
It's a plain bug that was introduced in 3.7.1.
It's on our todo list...

**Comment by JdeH** [comment]
Fixed, committed, released as 3.7.6
Feel free to reopen if you still encounter problems with this
